src/EnergyPlus/ConfiguredFunctions.in.cc

```diff
@@ -57,7 +57,7 @@ fs::path configured_source_directory()
 
 fs::path configured_build_directory()
 {
-    return ("${CMAKE_BUILD_DIR}");
+    return ("${CMAKE_BINARY_DIR}");
```

Comment:
Unrelated but was incorrect. I tried to use that to use an actual eso & mtr file instead of the open_as_stringstream to see if the test could work fast enough, but no.

Reply:
Wow.  Good catch.